subprojects/logging/src/main/java/org/gradle/internal/logging/sink/OutputEventRenderer.java

```diff
@@ -61,6 +65,8 @@
  */
 @ThreadSafe
 public class OutputEventRenderer implements OutputEventListener, LoggingRouter {
+    static final long UPDATE_NOW_FLUSH_INITIAL_DELAY_MS = 5000L;
```

Comment:
Having an initial delay of 5 seconds now seems kinda weird since the executor is no longer triggered by output. I think we should make the initial delay the same as the flush period.